src/components/index.tsx

```diff
@@ -0,0 +1 @@
+export {default as SidebarItem} from './nav/sidebarItem'
```

Comment:
Don't see the need of this if it's just one file.

Reply:
It actully not used. Deleted it.